rails/ordinals/es-419.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+{
+  "es-419": {
+    number: {
+      nth: {
+        ordinals: lambda do |_key, number:, **_options|
```

Comment:
:number is not a keyword argument: https://github.com/rails/rails/commit/1e4e0935bcf14cd5ab7c384ea9e3b91c4a8387b7